### ISSUE TITLE
Players unable to offline forceload

### ISSUE THREAD

**Issue by JayAitch** [thexaero/open-parties-and-claims#528]
Having issues allowing players to offline forceload or change the option dispite (what i think) to be the cofigurations set up. Currently using version `open-parties-and-claims-forge-1.19.2-0.22.0` on the server/client.

Offline forceloading is disabled, and isnt modifyable by the players on my server.
I have added `claims.forceload.offlineForceload` to `[server]/config/openpartiesandclaims-server.toml` to the string arrays at `playerConfigurablePlayerConfigOptions` and `opConfigurablePlayerConfigOptions`.

I have also modified `offlineForceload = true` within `[server]/config/openpartiesandclaims-default-player-config.toml`.

Is there something im missing to allow players to forceload whilst offline?

**Comment by thexaero** [comment]
Did you stop the server before editing `[server]/config/openpartiesandclaims-server.toml`?
Also, please use the mod's in-game UI screen to edit anything other than the main server config.

**Comment by JayAitch** [comment]
Yup, server was off,  both .toml files where updated after booting server. 

My understanding of `[server]/config/openpartiesandclaims-default-player-config.toml` is that its a default configuration for players allowing me to modify their defaults when they load, is that incorrect?

Is there a way of editing the default player config server side?

Thanks for the help :)

**Comment by thexaero** [comment]
> Yup, server was off, both .toml files where updated after booting server.

I don't think I understand this.

All "player configs" including wilderness, server claims and expired claims should be edited with the in-game UI screen that you can open with a key binding. It's set to ' by default.

**Comment by JayAitch** [comment]
Well when I modify that file to enable the option to be edited on players' clients. The option is disabled in the menu.

**Comment by thexaero** [comment]
You're either breaking the syntax or trying to change the file while the server is running.
